en/TOC.md

```diff
@@ -8,10 +8,7 @@
 + Introduction
   - [Overview](tidb-operator-overview.md)
   - [TiDB Operator v1.1 Notice](notes-tidb-operator-v1.1.md)
-+ Get Started
-  - [kind](deploy-tidb-from-kubernetes-kind.md)
-  - [GKE](deploy-tidb-from-kubernetes-gke.md)
-  - [Minikube](deploy-tidb-from-kubernetes-minikube.md)
+- [Get Started](get-started.md)
```

Comment:
From the following structures, it seems that we should have two sub-items here:
+ Get started
  - kind & minikube
  - gke

Reply:
I think that makes the structure a lot more confusing. I like the idea of having this single, consolidated Get Started page that gives users an overview of all the options.